### PR TITLE
Updating colors, elevation, illustrations, icons and typography topics

### PR DESCRIPTION
**Pull request by StefanIvanov**
Adding first changes to colors topic

### CODE REVIEW

**Review by radomirchev — APPROVED** [review]

File: en/style/colors.md [hunk 1]
```diff
@@ -1,93 +1,64 @@
...
-
-Upon changing the bottom fill, you will see the whole primary palette instantly update. The same palette generation is available for the secondary color.
-
-<img class="responsive-img" src="../images/colors_generation.png" srcset="../images/colors_generation@2x.png 2x" />
-
-With the introduction of Library Styles in Sketch and our support for Shared Color Styles there is one more step that you need to do i.e. to update all respective color styles that have changed with the selection of a new primary/secondary color. In order to do that upon selecting an individual color element, you should be able to observe the following inconsistency.
-
-<img class="responsive-img" src="../images/colors_styles_update.png"/>
+The **Indigo.Design System** version 3 introduces the concept of series palettes with two collections of ten colors each optimized for various data visualization scenarios. Both palettes ensure good contrast between adjacent series colors so that the palette can be used e.g. to colorize ranges of a gauge. One of the palettes is further optimized to be 100% inclusive and facilitate easy comprehension by people with color vision defficiencies.
```
Comment (radomirchev) [inline en/style/colors.md:13]: The **Indigo.Design System** version 3 - is that official version of Indigo?

**Review by simeonoff — APPROVED** [review]

File: en/style/colors.md (hunk 1 above)
Comment (StefanIvanov) [reply en/style/colors.md:13]: YES

File: en/style/typography.md [hunk 2]
```diff
@@ -1,93 +1,79 @@
...
+Use these when creating intricate layouts for articles, blog posts etc. The Text Styles in the **Indigo.Design System** are meant to help you define a consistent theme and style for all the texts in your designs.
 
-<img class="responsive-img" src="../images/typography_colors.png" srcset="../images/typography_colors@2x.png 2x" />
-
-## Component specific typography
-
-Components, such as [Avatar](../components/avatar.md) and [Hyperlink](../components/hyperlink.md) use component-specific Typography to accommodate the specifics of the styling used by the respective component e.g. the Avatar needs a larger variety of colors, while the Hyperlink comes with an underlined text.
+> [!Warning]
+> There is also a collection of `Internal` styles used within the Components for validation, special cases such as the label of a Button, and data-dense scenarios like the Grid. These styles are not supposed to be inserted in your interface directly, but exist to assure typograpic consistency for the Components within which they are used.
```
Comment (YoannaIvanova) [inline en/style/typography.md:23]:
```suggestion
> There is also a collection of `Internal` styles used within the Components for validation, special cases such as the label of a Button, and data-dense scenarios like the Grid. These styles are not supposed to be inserted in your interface directly but exist to assure typographic consistency for the Components within which they are used.
```

File: en/style/typography.md [hunk 3]
```diff
@@ -1,93 +1,79 @@
...
 
-<img class="responsive-img" src="../images/typography_specific.png" srcset="../images/typography_specific@2x.png 2x" />
+## Typography Colors
 
-## Changing the typeface
+Typography comes in multiple preset colors such as: `active` corresponding to `grays.900`, `inactive` corresponding to `grays.700`, `disabled` corresponding to `grays.500`, `white` and `secondary`. There are also type scales with special colors for strings that need to be in `success`, `warn`, and `error` colors, as well as a some additional nuances used by the Indigo.Design Components.
```
Comment (YoannaIvanova) [inline en/style/typography.md:27]:
```suggestion
Typography comes in multiple preset colors such as `active` corresponding to `grays.900`, `inactive` corresponding to `grays.700`, `disabled` corresponding to `grays.500`, `white` and `secondary`. There are also type scales with special colors for strings that need to be in `success`, `warn`, and `error` colors, as well as some additional nuances used by the Indigo.Design Components.
```

**Review by YoannaIvanova — CHANGES_REQUESTED** [review]

**Review by YoannaIvanova — APPROVED** [review]

**Pull request merged**